webclient/html/raflow.css

```diff
@@ -387,3 +387,10 @@ section[name="RABGInfoForm"] .w2ui-form-box .w2ui-field > div {
 .ra-form-component#rentables .grid-container {
     height: calc(100% - 90px);
 }
+
+.ra-people-section-title {
+    padding: 3px;
+    font-weight: bold;
+    color: #777;
+    padding-top: 20px;
```

Comment:
better could be written: `padding: 20px 3px 3px 3px;`. It follows as top, right, bottom, left sides. :)

Reply:
ok.